command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Fri, Jul 29, 2016 at 09:15:54AM -0700, Mike Brown wrote:
> It's a matter of defining terms and then being consistent with their
> use.  Runtime is a broad term.  Runtime-CLI is more specific.

We don't use “Solaris runtime” in config-solaris.md or “Linux runtime”
in config-linux.md.  I don't see why we should use “Runtime-CLI” in
command-line-interface.md.  Any given runtime can choose to be
compliant with as much or as little of any of these portions of
runtime-spec as it wants.  I think it is unlikely that any one runtime
will be compliant with *all* of runtime-spec.

The Compliance Working Group / Trademark Board will decide which
subsets of runtime-spec are certifiable and which are not.  I expect
that will currently include “all of the command line API” (whichever
repository it ends up in, assuming it lands under the OCI at all)
because [1].  If/when the OCI lands an additional API, the subsets
that include at least one of the APIs will probably be certifiable as
well [2].  Currently “which MUST subsets are certifiable” seems
largely orthogonal to “how those MUSTs are allocated between specs”.
Once we have clear feedback from the Compliance Working Group /
Trademark Board, it may make sense to partition the runtime specs into
separate repositories to align with their certifiable subsets.

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72694603
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72701237